### DISCUSSION TITLE
Pass additional data from getStaticPaths to getStaticProps

### DISCUSSION THREAD

**Discussion by arthurvi** [vercel/next.js#10977] [Help]
Hi,

Is it possible to return additional data in the `getStaticPaths()` response? And read that in `getStaticProps(context)`

For example 
```
{ 
  paths: [{ params: { slug: ['content'], data: 'extra' } }], 
  fallback: false 
}
```

Currently the `context` parameter in `getStaticProps()` is just 

```
{ params: { slug: [ 'content' ] } }
```

We would like to fetch data from our CMS by ID in `getStaticProps`, but show a different slug.

**Comment by kunal-thinlimbs** [comment]
is it possible yet

**Comment by zeing** [comment]
wanna too

**Comment by balazsorban44** [comment]
Might be a duplicate of https://github.com/vercel/next.js/discussions/11272?
